app/assets/javascripts/application.js

```diff
@@ -12,5 +12,5 @@
 //
 //= require jquery
 //= require jquery_ujs
-//= require turbolinks
+// require turbolinks
```

Comment:
what does turbolinks do? advantage? 

if you had to fix the original issue of javascript not rendering how would you go about it?